frontend-new/src/userPreferences/UserPreferencesService/userPreferences.service.ts

```diff
@@ -109,7 +116,7 @@ export default class UserPreferencesService {
       expectedContentType: "application/json",
     });
     const responseBody = await response.text();
-    return this.parseJsonResponse(responseBody, errorFactory);
+    return this.parseJsonResponse(responseBody, user_preferences.user_id, errorFactory);
```

Comment:
I see. cause the payload doesnt have a userId. this will probably change back when  we change the endpoint to respond with an object that has the userId... right?

Reply:
yeah sure.